pkg/printers/externalversion/describe.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package internalversion
+package externalversion
```

Comment:
@juanvallejo This PR is a great start. But since this file still has numerous dependencies on internal types, I'm not sure it is a good idea to move this file into `pkg/kubectl` yet. Moving this file into `pkg/kubectl` will make it more difficult to achieve `kubectl` independence. I suggest we do NOT move this file into `pkg/kubectl` until we've cleaned this file up more by removing the remaining internal dependencies.